tests/robot-tests/run_tests.py

```diff
@@ -145,12 +145,12 @@ def authenticate_user(user, email, password, clear_existing=True):
             cookie_file.unlink(True)
 
         if local_storage_file.exists() and cookie_file.exists():
-            print(f'Getting {user} authentication information from local files... ')
+            print(f'Getting {user} authentication information from local files... ', flush=True)
```

Comment:
Why do we need the flushes?

Reply:
Otherwise, all the prints appear at once when the robot tests start running proper, rather than appearing as they happen.

Reply:
I've never really noticed that. Is it a Windows thing?

Reply:
If it's not happening to you, then I guess so. It is weird why it isn't automatically triggering the flush of print statements to the terminal.